<launch>
	<rosparam command="load" file="$(find mpc_path_tracking)/config/mpc_params.yaml" />
	<node name="mpc_path_tracking_node" pkg="mpc_path_tracking" type="mpc_path_tracking" output="screen">
	</node>

<!-- <node pkg="tf2_ros" type="static_transform_publisher" name="base_to_map" args="0 0 0 0 0 0 map base_link" /> -->


</launch>




